### DISCUSSION TITLE
Simple syntax for css variables as values

### DISCUSSION THREAD

**Discussion by manuelmeister** [tailwindlabs/tailwindcss#9785] [Ideas]
We could create a simplified syntax for css variables:

`w-(size)` instead of `w-[var(--size)]`

**Comment by arkadiuszbachorski** [comment]
I also thinked of this, but with different syntax: `w-[--size]`. It might look funky a first glance, but:

* `--` is still there, which clearly stands for CSS Variables
* we keep `[]`, like for any regular arbitrary value
* parentheses (`()`) are reserved for arguments in functions and selectors, I'd not use them in a different context.

What do you think?

**Reply by adamwathan** [reply]
Had a similar thought after reading this discussion a few days ago! 😅

<img width="891" alt="image" src="https://user-images.githubusercontent.com/4323180/201716445-bc098882-f531-41c8-9d42-1dc39ae64210.png">

Will explore this for sure, thanks for the suggestion @manuelmeister 👍

**Comment by adamwathan** [comment]
Opened a PR for this here! https://github.com/tailwindlabs/tailwindcss/pull/9880

Thanks for the suggestion, I think this is a really nice improvement 👍
